backend/src/nodes/impl/cas.py

```diff
@@ -37,7 +37,7 @@ def create_cas_mask(img: np.ndarray, kernel, bias: float = 2) -> np.ndarray:
     min_l = cv2.erode(l, kernel)
     max_l = cv2.dilate(l, kernel, dst=l)
     min_d = np.minimum(1.0 - max_l, min_l, out=min_l)  # type: ignore
-    max_l += 1e-8  # type: ignore
+    max_l = np.add(max_l, 1e-8)
```

Comment:
This fix is not correct. For context:

```py
    l = _luminance(img)
    min_l = cv2.erode(l, kernel)
    max_l = cv2.dilate(l, kernel, dst=l)
    min_d = np.minimum(1.0 - max_l, min_l, out=min_l)  # type: ignore
    max_l = np.add(max_l, 1e-8)
```

The issue is that this code assumes that `l` is its own array (and not just a slice of the (readonly) input image). The fix is incorrect because `cv2.dilate(l, kernel, dst=l)` apparently just ignores that `l`'s underlying array is readonly, so we still modify the input image.

The correct fix is:

```diff
-max_l = cv2.dilate(l, kernel, dst=l)
+max_l = cv2.dilate(l, kernel)
```